docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,183 @@ Interventions
 Vitamin A Fortification
 ~~~~~~~~~~~~~~~~~~~~~~~
 
+Effect Size
+^^^^^^^^^^^
+
+**Research Considerations**
+
+In this model, the vitamin A fortification intervention affects the 
+**prevalence of vitamin A deficiency**. The effect size for this intervention 
+was obtained from a Cochrane review performed by Hombali et al. (2019) on the 
+fortification of staple foods with vitamin A for vitamin A deficiency. 
+Notably, the relative risk for vitamin A foritification on vitamin A 
+deficiency from this review only included data from two randomized controlled 
+trials and the authors of the review assessed the certainty of the evidence to 
+be "very low" (Hombali et al. 2019). The relative risk of vitamin A deficiency 
+prevalence among the population exposed to vitamin A fortification relative to 
+the population unexposed to vitamin A fortification from this review was 
+**0.45 (95% CI: 0.19 - 1.05)**.
+
+Therefore, we conducted a supplementary analysis of the effect of the 
+intervention by pooling the RCT studies from the Cochrane review with studies 
+included in the systematic review and meta-analysis performed by Keats et al. 
+(2019). Notably, none of the studies identified from the Keats et al. (2019) 
+review *directly* reported measures of relative risk of vitamin A deficiency 
+prevalence among the population exposed to vitamin A fortification relative to 
+the population unexposed to vitamin A fortification. Therefore, we manually 
+calculated this value based on data reported in study tables and figures, 
+which required visual approximations of certain values. Notably, when this 
+supplementary meta-analysis was performed, the resulting relative risk was 
+calcualted as **0.43 (95% CI: 0.28 - 0.65)**. However, when limited to sugar 
+and oil vehicles for the vitamin A forticant, the relative risk was **0.36 
+(95% CI: 0.26 - 0.50)**. These two supplementary meta-analyses are represented 
+in the forest plots below. 
+
+.. image:: vitamin_a_meta.png
+
+.. image:: vitamin_a_meta_sugar_oil.png
+
+While the supplementary meta-analysis shown above contains more studies and 
+data than the Cochrane review, it relies on results that were not directly 
+reported in the individual studies (and in some cases visaully estimated 
+values). **Therefore, we will conservatively use the results from the Cochrane 
+review, with increased certainty in the results based on the confirmatory 
+results from the supplementary meta-analysis.**
+
+Notably, all of these studies included in the supplementary analysis were 
+conducted among children and study locations included Guatemala, South Africa, 
```

Comment:
Typo? There are two "included"s in this sentence... I'm not sure exactly what it's trying to say.

Reply:
I don't think it's a typo but maybe could be more clear. I mean to say that the study populations for each of the studies that we used for the supplementary meta-analysis were children from one of the countries in that list. Is that more clear?

Reply:
Just noting that the studies looked specifically at children and listed the countries so that we could see they were largely LMICs, which supports the theory of generalizability to our study populations.

Reply:
Oh, maybe you're just missing a comma before the and: "children, and study locations..."

Reply:
Ok. I will update this.